tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/ant/CleanOutputDirectoryTask.java

```diff
@@ -86,7 +90,6 @@ public static final class Retain extends Task {
 
         // Don't use "Path" for the argument type because that always makes an absolute path (e.g.
         // relative to the working directory for the Ant task). We want relative paths.
```

Comment:
optional: remove references to and/Ant in comments?

Reply:
Done